service/controller/v26/resource/configmap/desired.go

```diff
@@ -89,15 +90,16 @@ func (r *Resource) newConfigMaps(customResource v1alpha1.KVMConfig) ([]*apiv1.Co
 // serves as structure being injected into the template execution to interpolate
 // variables. prefix can be either "master" or "worker" and is used to prefix
 // the configmap name.
-func (r *Resource) newConfigMap(customResource v1alpha1.KVMConfig, template string, node v1alpha1.ClusterNode, prefix string) (*apiv1.ConfigMap, error) {
-	var newConfigMap *apiv1.ConfigMap
+func (r *Resource) newConfigMap(customResource v1alpha1.KVMConfig, template string, node v1alpha1.ClusterNode, prefix string) (*corev1.ConfigMap, error) {
+	var newConfigMap *corev1.ConfigMap
 	{
-		newConfigMap = &apiv1.ConfigMap{
+		newConfigMap = &corev1.ConfigMap{
 			ObjectMeta: apismetav1.ObjectMeta{
 				Name: key.ConfigMapName(customResource, node, prefix),
 				Labels: map[string]string{
-					"cluster":  key.ClusterID(customResource),
-					"customer": key.ClusterCustomer(customResource),
```

Comment:
Are you sure we can simply drop these now? I cannot tell you what breaks now. We either need to understand this better or do a graceful migration. The latter could mean to keep the legacy labels, have a TODO issue and drop the legacy labels when `v26` is gone.

Reply:
You're right, I think we can keep these labels at the moment and later drop it. I will address it afterward.